anchors/anchor_test.go

```diff
@@ -12,11 +12,15 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var ctx = map[string]interface{}{}
+var cfg *config.Configuration
```

Comment:
I would love to avoid adding these, they create problems for running tests in parallel.